public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -237,31 +238,23 @@ figure.image-display
     we were conditionally including or excluding a big chunk of HTML with many data bindings.
 
 :marked
-  Try it out. We have four items in the array so the message should appear.
+  Try it out. Because the array has four items, the message should appear.
   Go back into `app.component.ts` and delete or comment out one of the elements from the hero array.
   The browser should refresh automatically and the message should disappear.
 
-  Play with it.
-
 .l-main-section
 :marked
   ## Summary
-  Now we know how to
-  - use **interpolation** with the double curly braces to display a component property,
-  - use **`NgFor`** to display a list of items,
-  - use a TypeScript class to shape the model data for our component and display properties of that model,
-  - use **`NgIf`** to conditionally display a chunk of HTML based on a boolean expression.
+  Now we know how to use:
+  - **interpolation** with double curly braces to display a component property
+  - **`NgFor`** to display a list of items
+  - a TypeScript class to shape the **model data** for our component and display properties of that model
+  - **`NgIf`** to conditionally display a chunk of HTML based on a boolean expression
 
-  Our final code:
+  Here's our final code:
 
 +makeTabs(`displaying-data/ts/app/app.component.ts,
            displaying-data/ts/app/hero.ts,
            displaying-data/ts/app/boot.ts`,
            'final,,',
            'app/app.component.ts, app/hero.ts, boot.ts')
-
-.l-main-section
-:marked
-  ## Next Steps
```

Comment:
I've been including a "Next Steps" section in the introductory chapters that are on the learning path described in the introduction.

I don't have such a section for the more "advanced"  topics.

Do you have a better idea?

Naomi proposed separating the learning path chapters into a "Basics" section with sequence number and moving all other chapters to an "Advanced" in alphabetical order.  That would make this moot.